### PR TITLE
chore: update analytics tests

### PR DESCRIPTION
**Pull request by jimhunty**
Following up from #954, this updates the analytics tests in our page components to follow the same pattern.

### CODE REVIEW

File: packages/author-profile/__tests__/shared.js [hunk 1]
```diff
@@ -102,18 +105,10 @@ export default () => {
...
-        component: "Page",
-        action: "Viewed",
-        attrs: expect.objectContaining({
-          articlesCount: 20,
-          authorName: mockAuthor.name,
-          page: 1,
-          pageSize: 3
-        })
-      })
+    const call = reporter.mock.calls[0][0];
```
Comment (colincclark) [inline packages/author-profile/__tests__/shared.js]: this overlaps with https://github.com/newsuk/times-components/pull/953

**Comment by times-tools** [comment]
If you use Expo, view our components by scanning this qr code: <br>  <br> This has been made possible through [Fructose](https://github.com/newsuk/fructose)

File: packages/author-profile/__tests__/shared.js [hunk 2]
```diff
@@ -102,18 +105,10 @@ export default () => {
...
+
+    expect(call).toMatchSnapshot(
+      "5. Send analytics when rendering an Auhtor Profile page"
```
Comment (colincclark) [inline packages/author-profile/__tests__/shared.js]: typo: `Auhtor` -> `Author`

**Comment by times-tools** [comment]
If you use Expo, view our components by scanning this qr code: <br>  <br> This has been made possible through [Fructose](https://github.com/newsuk/fructose)

**Pull request merged**